lib/bolt/cli.rb

```diff
@@ -114,10 +114,11 @@ def parse
       @config = if options[:configfile]
                   Bolt::Config.from_file(options[:configfile], options)
                 else
-                  project = if options[:boltdir]
-                              Bolt::Project.create_project(options[:boltdir])
+                  boltdirpath = options[:boltdir] || Dir.pwd
+                  project = if (boltdirpath + Bolt::Project::BOLTDIR_NAME).directory?
+                              Bolt::Project.create_project(boltdirpath + Bolt::Project::BOLTDIR_NAME, 'embedded')
                             else
-                              Bolt::Project.find_boltdir(Dir.pwd)
```

Comment:
This now removes the only use of the `find_boltdir` method. It will cause us to _never_ use the user- or system-level project.

I think the logic needs to be:
a) if `options[:boltdir]` is set and contains `Boltdir`, use `options[:boltdir] + Boltdir`
b) if `options[:boltdir]` is set and doesn't contain `Boltdir`, use `options[:boltdir]`
c) otherwise call `find_boltdir(Dir.pwd)`